domains/filmflu.is-a-dev.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Film Flu Web Platform",
+    "repo": "https://github.com/dherediat97/FilmFlu",
+    "owner": {
+        "username": "dherediat97",
+        "email": "dherediat97@gmail.com",
+    },
+    "record": {
+        "CNAME": "dherediat97.github.io/FilmFlu"
```

Comment:
Invalid CNAME record, cannot contain slashes.

Reply:
So, How do a redirect of a github pages? Thanks

Reply:
You could use URL or just the first part of the CNAME.